docs/src/content/docs/pt/challenges/angular/16-di.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 🔴 Dominando Injeção de Dependência
+description: Desafio 16 é sobre dominar como injeção de dependência funciona
+author: thomas-laforge
+contributors:
+  - tomalaforge
+  - tomer953
```

Comment:
you need to replave the contributors list with your name.

Reply:
Done!